### PR TITLE
Rate goal progress template

### LINKED ISSUE #18: Template for rating progress
- [x] Back button.
- [x] The title of the goal
- [ ] Question to rate the progress.
- [ ] Seed / flower image.
- [x] A scale.
- [x] Save button.

### LINKED ISSUE #19: Make the scale interactive on rate progress page.
- [ ] cool slider icon
- [ ] update value with javascript
- [ ] add numbers to scale (0 and 10 at least)

### PR DESCRIPTION
**Pull request by emilyb7**
relates to #12 
resolved #18 
partially resolves #19

### CODE REVIEW

File: public/styles/_add-goal.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    height: 6%;
+    display: flex;
+    align-items: center;
+}
+
+
+.back-button {
+  margin-left: 5px;
+  flex-direction: row;
+  // height: 28px;
```
Comment (des-des) [inline public/styles/_add-goal.scss]: :fire:

File: public/styles/_add-goal.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  width: 97%;
+  align-self: center;
+}
+
+.new-goal-label {
+  font-size: 1.3em;
+}
+
+.new-goal-input {
+  // height: 40px;
```
Comment (des-des) [inline public/styles/_add-goal.scss]: :fire:

File: public/styles/_colors.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+$yellow: #fcfc62;
```
Comment (des-des) [inline public/styles/_colors.scss]: :tada:

File: public/styles/style.css [hunk 4]
```diff
@@ -0,0 +1,277 @@
+.page, .add-goal-page, .rate-progress-page {
```
Comment (des-des) [inline public/styles/style.css]: hmm

Comment (des-des) [inline public/styles/style.css]: This file again ...

File: public/styles/style.css.map [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (des-des) [inline public/styles/style.css.map]: And this one?

**Comment by des-des** [comment]
This is why commits compiled code is bad: 
image

I might be easiest to respond to comments before you fix the merge.. @emilyb7

File: public/styles/style.css (hunk 4 above)
Comment (emilyb7) [inline public/styles/style.css]: omg it won't go away 🙈

**Comment by emilyb7** [comment]
@des-des can ok now?

**Comment by bradreeder** [comment]
relate all commits to their issues please, see my comment on this pr for rationale https://github.com/CYPIAPT-LNDSE/welcome-to-camhs/pull/28#issuecomment-276450618

^_^

**Pull request merged**